.travis.yml

```diff
@@ -21,23 +21,20 @@ env:
     - PIP_WHEEL_DIR=$HOME/.cache/pip/wheels
     - PIP_FIND_LINKS=file://$HOME/.cache/pip/wheels
 
-python:
-    - 2.6
-    - 2.7
-
+  matrix:
+    - TOX_ENV=py26
+    - TOX_ENV=py27
```

Comment:
I'm surprised this works. I guess it's because all linux builds have python installed. 

Normally you do something like this:
```
language: python
env:
  - TOX_ENV=py26
  - TOX_ENV=py27

script:
  - tox -e $TOX_ENV
```